drivers/video/video_mcux_csi.c

```diff
@@ -396,6 +396,8 @@ static int video_mcux_csi_init(const struct device *dev)
 	struct video_mcux_csi_data *data = dev->data;
 	int err;
 
+	data->dev = dev;
```

Comment:
this change also seems unrelated to the rest of the PR also, typo in commit message (independant -> independent)